docs/source/extending/extending_models.rst

```diff
@@ -0,0 +1,240 @@
+Extending the Interaction Models
+================================
+Let's assume you have invented a new interaction model,
+e.g. this variant of :class:`pykeen.models.DistMult`
+
+.. math::
+
+    f(h, r, t) = <h, \sigma(r), t>
+
+where :math:`h,r,t \in \mathbb{R}^d`, and :math:`\sigma` denotes the logistic sigmoid.
+
+Picking a Base Class
+--------------------
+From the reference documentation on base models (:mod:`pykeen.models.base`), we can see that
+:class:`pykeen.models.base.EntityRelationEmbeddingModel` is a good candidate for a base class
+since we want to have embeddings for entities *and* relations.
+
+Implementing ``score_hrt()``
+----------------------------
+The only implementation we have to provide is of the `score_hrt` member function:
+
+.. code-block:: python
+
+    from pykeen.models.base import EntityRelationEmbeddingModel
+
+    class ModifiedDistMult(EntityRelationEmbeddingModel):
+        def score_hrt(self, hrt_batch):
+            # Get embeddings
+            h = self.entity_embeddings(  hrt_batch[:, 0])
```

Comment:
Remove additional whitespace?

Reply:
Fixed in a6afff03

Reply:
I thought this might make it more readable but I was wrong 😆